checkstyle.xml

```diff
@@ -208,5 +208,6 @@
             <property name="exceptionVariableName" value="expected"/>
         </module>
         <module name="CommentsIndentation"/>
+        <module name="UnusedImports"/>
```

Comment:
the spring boot template app has this rule among others, I would suggest getting the rules from there instead of this one rule.
It also has an updated checkstyle version

Reply:
ah. then this pr is irrelevant :smiley: